include/openssl/x509.h

```diff
@@ -547,7 +548,7 @@ DECLARE_ASN1_FUNCTIONS(X509_NAME_ENTRY)
 
 DECLARE_ASN1_FUNCTIONS(X509_NAME)
 
-int X509_NAME_set(X509_NAME **xn, const X509_NAME *name);
+int X509_NAME_set(const X509_NAME **xn, const X509_NAME *name);
```

Comment:
Note for **

Reply:
Also here (adding `const ` to the types of function parameters) I see no compatibility issue.

Reply:
I've reverted this change.